### PR TITLE
doc: Debugging Cypress on CI using artifacts

### PR DESCRIPTION
**Pull request by Sidsector9**
### Description
Adds a small brief for debugging Cypress on CI using Cypress' screenshot and video recording capabilities in tandem with Github artifacts.

### CODE REVIEW

File: _includes/markdown/Testing.md [hunk 1]
```diff
@@ -44,3 +44,26 @@ cy.visit( `/wp-admin` );
...
+Configuring Cypress to capture screenshots and videos of the tests is pretty straightforward. And, unless explicitly configured, Cypress will store the screenshots and videos under the `cypress/screenshots` and `cypress/videos` directories by default.
+
+We configure the Upload Artifact Action to build an artifact out of these 2 directories, which will be later available to us for download and inspection.
+
+The simplest configuration is as follows:
+
+```yaml
+# actions.yml
+
+- name: Upload RSA artifcats
```
Comment (jeffpaul) [inline _includes/markdown/Testing.md]: Let's remove the "RSA" reference to make this more generic, otherwise this is great to merge, thanks!

**Review by jeffpaul — CHANGES_REQUESTED** [review]
One small change, otherwise good to go!

File: _includes/markdown/Testing.md (hunk 1 above)
Comment (dinhtungdu) [reply _includes/markdown/Testing.md]: I agree. And also a small typo issue here ; )

File: _includes/markdown/Testing.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ The simplest configuration is as follows:
 ```yaml
 # actions.yml
 
-- name: Upload RSA artifcats
+- name: Upload artifcats
```
Comment (dinhtungdu) [inline _includes/markdown/Testing.md]: Small typo issue here.
```suggestion
- name: Upload artifacts
```

**Pull request merged**